CHANGELOG.md

```diff
@@ -1,16 +1,24 @@
 ## Unreleased
 
+## 1.3.0 (April 6, 2023)
```

Comment:
I could also see this being a patch release (1.2.1) since it's mostly dependencies and a bug fix.

Reply:
Works for me.